### ISSUE TITLE
Why not add this project to bower/npm ?

### ISSUE THREAD

**Issue by hxlniada** [alexklibisz/angular-dual-multiselect-directive#1]
Very useful, Why not add this project to bower/npm ?

**Comment by alexklibisz** [comment]
True! Glad you found it useful. It's something I've been meaning to refine a bit and then I'll get it up on one or both of those.

Thanks for the feedback!

Alex

Sent from my HTC One on the Verizon Wireless 4G LTE network

----- Reply message -----

**Comment by yeahoffline** [comment]
+1 for bower

**Comment by alexklibisz** [comment]
I'll clean it up and submit to bower this weekend. Thanks for the positive support.
